src/nvim/event/libuv_process.c

```diff
@@ -21,6 +21,7 @@ int libuv_process_spawn(LibuvProcess *uvproc)
   uvproc->uvopts.file = proc->argv[0];
   uvproc->uvopts.args = proc->argv;
   uvproc->uvopts.flags = UV_PROCESS_WINDOWS_HIDE;
+  uvproc->uvopts.flags |= UV_PROCESS_WINDOWS_VERBATIM_ARGUMENTS;
```

Comment:
This was removed back in https://github.com/neovim/neovim/pull/5280#issue-174549578 is the goal to do quoting in neovim instead of relying in libuv?